modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -460,6 +460,40 @@ definition {
 		}
 	}
 
+	@description = "LPS-182505 - Verify that it's possible to delete an user notification template"
+	@priority = 3
+	test CanDeleteUserTemplate {
+		task ("Given a User Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+				ObjectNotifications.addNewNotificationTemplate(
+					description = "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+					name = "User Notification Template",
+					notificationType = "User Notification",
+					recipientType = "User",
+					subject = "Subject content",
+					userName = "Test");
+
+				PortletEntry.save();
```

Comment:
We should do this via API instead, also your indentation is off for a few of the macros

Reply:
Our API macro for templates just work to email templates. Should I add it on this PR or do in the future?